sources/packages/backend/apps/api/src/route-controllers/application-offering-change-request/_tests_/e2e/application-offering-change-request.students.controller.getApplicationOfferingChangeRequest.e2e-spec.ts

```diff
@@ -0,0 +1,102 @@
+import { HttpStatus, INestApplication } from "@nestjs/common";
+import * as request from "supertest";
+import {
+  BEARER_AUTH_TYPE,
+  createTestingAppModule,
+  FakeStudentUsersTypes,
+  getStudentToken,
+  InstitutionTokenTypes,
+  getAuthRelatedEntities,
+  getStudentByFakeStudentUserType,
+} from "../../../../testHelpers";
+import {
+  saveFakeApplicationOfferingRequestChange,
+  createE2EDataSources,
+  E2EDataSources,
+  createFakeInstitutionLocation,
+  saveFakeApplication,
+} from "@sims/test-utils";
+import { InstitutionLocation, Student } from "@sims/sims-db";
+
+describe("ApplicationOfferingChangeRequestStudentsController(e2e)-getApplicationOfferingChangeRequest", () => {
+  let app: INestApplication;
+  let collegeFLocation: InstitutionLocation;
+  let student: Student;
+  let db: E2EDataSources;
+
+  beforeAll(async () => {
+    const { nestApplication, dataSource } = await createTestingAppModule();
+    app = nestApplication;
+    db = createE2EDataSources(dataSource);
+    const { institution: collegeF } = await getAuthRelatedEntities(
+      db.dataSource,
+      InstitutionTokenTypes.CollegeFUser,
+    );
+    collegeFLocation = createFakeInstitutionLocation({ institution: collegeF });
+    student = await getStudentByFakeStudentUserType(
+      FakeStudentUsersTypes.FakeStudentUserType1,
+      dataSource,
+    );
+  });
+
+  it("Should return the application offering change request details when provided with the application offering change request id.", async () => {
+    // Arrange
+    const application = await saveFakeApplication(db.dataSource, {
+      institutionLocation: collegeFLocation,
+      student,
+    });
+    const applicationOfferingChangeRequest =
+      await saveFakeApplicationOfferingRequestChange(db, {
+        institutionLocation: collegeFLocation,
+        application,
```

Comment:
not sure if we should be concerned about the application status in the TCs, Line 44 will create a submitted application, technically we need a completed application

Reply:
I can change it to a completed application.